rotkehlchen/history/price.py

```diff
@@ -351,3 +356,46 @@ def query_multiple_prices(
         )
 
         return assets_price
+
+    @staticmethod
+    def query_uniswap_position_price(
+            pool_token: EvmToken,
+            pool_token_amount: FVal,
+            to_asset: Asset,
+            timestamp: Timestamp,
+    ) -> Price:
+        """Return the uniswap position price at the given timestamp
+
+        Args:
+            pool_token: The token representing a Uniswap pool position
+            pool_token_amount: The amount of pool tokens
+            to_asset: The asset to convert the price
+            timestamp: The timestamp at which to get the price
+
+        Returns:
+            The price of the position at the given timestamp
+
+        May raise:
+            InputError: If the token's protocol is not Uniswap V2 or V3
+        """
+        evm_inquirer = Inquirer.get_evm_manager(chain_id=pool_token.chain_id).node_inquirer
+        block_number = evm_inquirer.get_blocknumber_by_time(timestamp)
```

Comment:
this can raise remoteError, the docstring is missing that error and also I don't see it being handled